browser/ui/BUILD.gn

```diff
@@ -650,13 +658,20 @@ source_set("ui") {
     ]
   }
 
+  if (!toolkit_views && !is_android) {
+    sources += [ "side_panel/ai_chat/ai_chat_side_panel_utils.cc" ]
```

Comment:
As the implementation is guarded, you can just put this on line 108 with the header.

Reply:
Done